e3db/base_crypto.py

```diff
@@ -10,6 +10,19 @@ class BaseCrypto:
     def get_mode(self):
         pass
 
+    @classmethod
+    def get_signature_version(self):
+        """
+        Gets the signature version that is supported for Crypto. 
+
+        Returns
+        -------
+        str
+            The current signature version is uuid-v5 format. 
+        """   
+
+        return 'e7737e7c-1637-511e-8bab-93c4f3e26fd9' 
```

Comment:
extract this to a named constant at (or near) the top of this file please

Reply:
added